src/globals.rs

```diff
@@ -82,6 +82,19 @@ pub enum NodeMode {
     Follower,
 }
 
+pub trait ModeString {
```

Comment:
This is not necessary. Just implement the `Display` trait. Or use the `strum` crate.